test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] test.md should not be committed to your Github repo, kindly consider removing it